compose/ui/ui-test-junit4/src/desktopTest/kotlin/androidx/compose/ui/test/ComposeUiTestTest.kt

```diff
@@ -126,6 +129,35 @@ class ComposeUiTestTest {
         }
     }
 
+    // See https://github.com/JetBrains/compose-multiplatform/issues/3117
+    // Note: this test is under desktopTest instead of skikoTest because there's no access to
+    // atomics in skikoTest
```

Comment:
There are a few expect/actual in the repo for `AtomicInt`: It's a typealias to `java.util.concurrent.atomic.AtomicInteger`on jvm and to `kotlin.native.concurrent.AtomicInt` on k/native

Reply:
It's `internal`, unfortunately. The `expect` is public (because of some bug), but the `actual`s are internal.